build/features.test.ts

```diff
@@ -30,6 +30,7 @@ const noScreenshotExceptions = new Set([
 	'selection-in-new-tab',
 	'submission-via-ctrl-enter-everywhere',
 	'click-outside-modal',
+	'scroll-to-current-file',
```

Comment:
No exceptions for this one, it needs a screenshot with an arrow pointing to the sidebar

Reply:
Thanks. I was about to remove it from exceptions once I figured out how to get a public link to the image.
The new screenshot attached to the bottom of the PR would be okay?

Reply:
> The new screenshot attached to the bottom of the PR would be okay?

Yes